test/integration/adapters/smtp_test.exs

```diff
@@ -16,15 +16,16 @@ defmodule Swoosh.Integration.Adapters.SMTPTest do
     ]
   end
 
-  defp config(Swoosh.Adapters.Muaua) do
-    [
-      relay: System.get_env("SMTP_RELAY"),
-      domain: System.get_env("SMTP_DOMAIN"),
-      auth: [
-        username: System.get_env("SMTP_USERNAME"),
-        password: System.get_env("SMTP_PASSWORD")
-      ]
-    ]
+  defp config(Swoosh.Adapters.Mua) do
+    relay = System.get_env("SMTP_RELAY", "localhost")
+    domain = System.get_env("SMTP_DOMAIN", "mua.local")
+    port = String.to_integer(System.get_env("SMTP_PORT", "1025"))
```

Comment:
I'm using env vars that work with Mailpit by default.

```sh
$ docker run -d --rm -p 1025:1025 -p 8025:8025 -e "MP_SMTP_AUTH_ACCEPT_ANY=1" -e "MP_SMTP_AUTH_ALLOW_INSECURE=1" --name mailpit axllent/mailpit
```

<img width="1582" alt="Screenshot 2024-12-11 at 15 59 59" src="https://github.com/user-attachments/assets/718c07e9-8a7a-4927-aceb-d64332e5e879">

<img width="1582" alt="Screenshot 2024-12-11 at 15 59 55" src="https://github.com/user-attachments/assets/8670b2e8-0dfb-4aba-9273-a85ac9c0d0bf">

I think we can also add it to CI unless there is already something like that. Here's how I use it in Mua:
- https://github.com/ruslandoga/mua/blob/99e905252201729e3d38ae643bce31977c550b0a/test/test_helper.exs#L15-L27
- https://github.com/ruslandoga/mua/blob/99e905252201729e3d38ae643bce31977c550b0a/.github/workflows/mix.yml#L15-L28

Reply:
Sounds good. We can do it in a later PR.